### DISCUSSION TITLE
ZkSync plugin not selecting my open contract to deploy

### DISCUSSION THREAD

**Discussion by capsimba** [Cyfrin/foundry-full-course-cu#2695] [Q&A]
Anyone else having the same issue where the Zksync cannot select an open contract for I to compile it and deploy.

Just installed the plug in following the steps in the course and no success whatsoever.

**Comment by EngrPips** [comment]
Hello @capsimba, are you working in the `default-workspace` and do you have your files in a `contracts` folder?

**Comment by Totiruzi** [comment]
Just like @EngrPips said you should have your files in a contracts folder.
